src/subdomains/supporting/pricing/pricing.module.ts

```diff
@@ -48,6 +52,8 @@ import { PricingServiceNew } from './services/pricing.service.new';
     PricingDeFiChainService,
     PricingCoinGeckoService,
     PricingDexService,
+    PricingFrankencoinService,
+    FrankencoinService,
```

Comment:
Don't provide the `FrankencoinService` in the PricingModule (will create a duplicate), it should be already imported by importing the `BlockchainModule`,